eslint.config.js

```diff
@@ -1,22 +1,32 @@
+// @ts-check
 import js from '@eslint/js'
 import ts from 'typescript-eslint'
+import tseslint from 'typescript-eslint'
 import svelte from 'eslint-plugin-svelte'
 import prettier from 'eslint-config-prettier'
+// @ts-expect-error "no types for eslint-plugin-storybook"
+import storybook from 'eslint-plugin-storybook'
 import globals from 'globals'
 
-/** @type {import('eslint').Linter.Config[]} */
-export default [
+export default tseslint.config(
   js.configs.recommended,
-  ...ts.configs.recommended,
+  ...ts.configs.recommendedTypeChecked,
   ...svelte.configs['flat/recommended'],
   prettier,
   ...svelte.configs['flat/prettier'],
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-argument, @typescript-eslint/no-unsafe-member-access
+  ...storybook.configs['flat/recommended'],
```

Comment:
not that we're using storybook yet, but let's include linting for it.

Reply:
We should be getting to that pretty soon